Modules/SwiftUISupport/SwiftUIContainerView.swift

```diff
@@ -0,0 +1,62 @@
+import JamitFoundation
+import SwiftUI
+import UIKit
+
+@available(iOS 13.0, *)
+/// Takes a swiftui view as a generic type and wraps it into a `StatefulView`.
+public class SwiftUIContainerView<ContentView: View>: StatefulView<EmptyViewModel> {
+    /// The UIViewController that holds the view hierarchy in which the `SwiftUIContainerView` is added as a subview
+    public weak var parentViewController: UIViewController?
+
+    private var hostingController: UIHostingController<ContentView>
+
+    /// Initializes a `StatefulViewRepresentable`
+    ///
+    /// - Parameters:
+    ///    - parentViewController: The UIViewController that manages the view hierarchy to which then `SwiftUIContainerView` is added.
+    ///    - contentView: The SwiftUI view to be wrapped
+    public init(parentViewController: UIViewController, contentView: ContentView) {
+        self.parentViewController = parentViewController
+        self.hostingController = .init(rootView: contentView)
+
+        super.init(frame: .zero)
+        isUserInteractionEnabled = false
```

Comment:
Maybe I'm again not getting something, but just wanted to make sure: Is this line intended to stay or is it there by mistake?